components/lock.rst

```diff
@@ -158,6 +151,49 @@ This component also provides two useful methods related to expiring locks:
 ``getExpiringDate()`` (which returns ``null`` or a ``\DateTimeImmutable``
 object) and ``isExpired()`` (which returns a boolean).
 
+
+The Owner of The Lock
+---------------------
+
+Locks that are acquired for the first time are owned by the process that acquired
+it. If you need to check whether the current process is (still) the owner of
+a process, you can use the ``isAcquired()`` method::
+
+    if ($lock->isAcquired()) {
+        // This process (still) owns the lock
+    }
+
+
+Because of the fact that some lock stores have expiring locks (as seen and explained
+above), it is possible for a process to lose the lock it acquired automatically.
+Here's an example for such a case::
```

Comment:
This sentence could be removed. And the previous sentence can end with „::“

Reply:
True, done :)